rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyParserVisitor.java

```diff
@@ -2595,6 +2653,7 @@ private TypeTree visitTypeTree(ClassNode classNode, boolean inferredType) {
 
         int saveCursor = cursor;
         Space fmt = whitespace();
+        // TODO: remove this? At least use `getModifiers` function
```

Comment:
Is this something you'd intended to pick up later? Or for this PR still?

Reply:
Later, I saw this line and I thought it was not good enough. At least just checking for `def` here is not enough.